samples/isilon_v230_k8s_121.yaml

```diff
@@ -196,6 +208,14 @@ spec:
       #  - key: "node.kubernetes.io/network-unavailable"
       #    operator: "Exists"
       #    effect: "NoExecute"
+      # - key: "node-role.kubernetes.io/master"
```

Comment:
Please confirm if this is tested as we didnt have it for helm charts too earlier